### PR TITLE
fix: cast jobId to string because job_id column indexed in database is varchar

### PR DESCRIPTION
**Pull request by NguyenUoc98**
Because job_id column indexed in database is varchar, if not cast $jobId to string, query is slow, not use index in query

Example with 2M rows in database:
Before
image

After
image

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
         {
-            "name": "romanzipp",
-            "email": "ich@ich.wtf",
-            "homepage": "https://ich.wtf"
+            "name": "Nguyễn Văn Ước",
+            "email": "uocnv.soict.hust@gmail.com",
+            "role": "Developer"
         }
     ],
     "require": {
```
Comment (romanzipp) [inline composer.json:13]: Please revert this change to the original composer.json

**Review by romanzipp — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (NguyenUoc98) [reply composer.json]: I reverted this commit

**Review by romanzipp — APPROVED** [review]
